src/components/HeaderWithCloseButton.js

```diff
@@ -138,26 +165,37 @@ class HeaderWithCloseButton extends Component {
                     styles.overflowHidden,
                 ]}
                 >
-                    {this.props.shouldShowBackButton && (
-                        <Tooltip text={this.props.translate('common.back')}>
-                            <Pressable
-                                onPress={() => {
-                                    if (this.props.isKeyboardShown) {
-                                        Keyboard.dismiss();
-                                    }
-                                    this.props.onBackButtonPress();
-                                }}
-                                style={[styles.touchableButtonImage]}
-                            >
-                                <Icon src={Expensicons.BackArrow} />
-                            </Pressable>
-                        </Tooltip>
+                    <View style={[styles.flexRow, styles.flex1]}>
```

Comment:
Explained below down here: https://github.com/Expensify/App/pull/18148/files/7d536701547969e16f47104bc55539703e9ad9ac#r1182854883

>  We want the AvatarWithDisplay to be grouped with the back button rather than in the center of the header.

Reply:
This causes regression: All RHN header styles are broken

![rhn header](https://user-images.githubusercontent.com/108292595/236373371-44b1fcc5-82bc-4ff5-8983-5e05b8a20e6d.png)